src/Microsoft.Health.Fhir.TemplateManagement/Client/OrasClient.cs

```diff
@@ -93,5 +106,27 @@ private async Task OrasExecutionAsync(string command, string orasWorkingDirector
                 throw new OrasException(TemplateManagementErrorCode.OrasTimeOut, "Oras request timeout");
             }
         }
+
+        private void AddFilePermissionInLinuxSystem(string fileName)
```

Comment:
Can we move this function to somewhere in common? It seems an useful function and used in unit tests as well.